src/main/java/org/aksw/simba/lemming/tools/PrecomputingValues.java

```diff
@@ -61,9 +64,15 @@ public static void main(String[] args) {
         if (args[0].equals("pg")) {
             USE_SEMANTIC_DOG_FOOD = false;
             USE_PERSON_GRAPH = true;
-        } else {
+            USE_LINKED_GEO = false;
+        } else  if (args[0].equals("swdf")){
             USE_SEMANTIC_DOG_FOOD = true;
             USE_PERSON_GRAPH = false;
+            USE_LINKED_GEO = false;
+        } else {
+        	USE_SEMANTIC_DOG_FOOD = false;
+            USE_PERSON_GRAPH = false;
+            USE_LINKED_GEO = true;
         }
```

Comment:
The same as in `GraphGenerationTest.java`. I would suggest a "defensive" programming and check whether linked geo data was given as argument or whether it is an unknown or mistyped dataset name.